app/assets/v2/css/profile.css

```diff
@@ -306,3 +310,84 @@
 .no-profile-textcolor {
   color: #666666;
 }
+
+#float_avatar_image{
+  z-index: 9999999;
+  margin-top: -140px;
+}  
+
+#float_avatar_image img{
+  height: 186px;
+  width: 186px;
+  border: 3px solid #FFFFFF;
+  box-shadow: 0 -2px 4px 0 rgba(0,0,0,0.50);
+}  
+
+#float_avatar_image.is_my_profile:hover img{
+  border: 3px solid #0fce7c;
+}  
+
+.is_my_profile{
+  cursor: pointer;
+}
+#avatar_edit{
+  text-align: center;
+  background-color: white;
+  border-radius: 50%;
+  opacity: 0.4;
+  filter: alpha(opacity=40);
+  color: black;
+  vertical-align: text-top;
+  height: 186px;
+  width: 186px;
+}
+#avatar_edit_text {
+  position: absolute;
+  top: -25px;
+  left: 55px;
+  color: black;
+  font-weight: bold;
+  z-index: 999999;
```

Comment:
Can be lower index to don't overlap modals, popovers, tooltips. Always try to use the index you just need instead of maximum index.